components/CandidateTestView/CandidateTestView.tsx

```diff
@@ -19,12 +19,12 @@ const CandidateTestView = (candidateTestViewProps: CandidateTestViewProps): JSX.
             </section>
             <section className={scss.textEditor}>
                 <TextEditorContainer 
-                                     height="50vh" 
-                                     width="50vw"
-                                     defaultText={candidateTestViewProps.test.testStarterCode}
+                     height="50vh" 
+                     width="50vw"
```

Comment:
I think this width should just be 100% - so that it fills its parent.
That way we can get rid of the confusing stuff on the parent element.